src/taco-remote-lib/ios/iosAppRunnerHelper.ts

```diff
@@ -22,24 +22,26 @@ import utils = require ("taco-utils");
 
 import UtilHelper = utils.UtilHelper;
 
-var proxyInstance: child_process.ChildProcess = null;
 var promiseExec: (...args: any[]) => Q.Promise<any> = Q.denodeify(UtilHelper.loggedExec);
 
 class IosAppRunnerHelper {
     public static startDebugProxy(proxyPort: number): Q.Promise<child_process.ChildProcess> {
-        if (proxyInstance) {
-            proxyInstance.kill("SIGHUP"); // idevicedebugserver does not exit from SIGTERM
-            proxyInstance = null;
+        if (!global.tacoRemoteLib) {
+            global.tacoRemoteLib = {};
```

Comment:
Why are we setting this? Is it not sufficient to add a guard to the statement below?